src/lib.rs

```diff
@@ -128,7 +128,7 @@ pub use crate::{constants::*, enums::*, js_collections::*, local::*, objects::*,
 /// use js_sys::{JsString, Reflect};
 /// use screeps::{game, prelude::*, Creep};
 ///
-/// let c = game::creeps().get(String::from("Bob")).unwrap();
+/// let c = game::creeps().get("Bob".into()).unwrap();
```

Comment:
`JsString::from` mirrors the `String::from` call that was here before, not that it matters, but it's interesting to change it.